src/AssetConverter.php

```diff
@@ -120,16 +127,41 @@ public function convert(string $asset, string $basePath): string
         return $asset;
     }
 
+    /**
+     * Allows you to set a command that is used to perform the asset conversion.
+     *
+     * @param string $key file extension
+     * @param array $value
+     *
+     * Example:
+     *
+     * $converter->setCommand('php', ['txt', 'php {from} > {to}']);
+     *
+     * @return void
+     */
     public function setCommand(string $key, array $value): void
```

Comment:
How about changing signature to `setCommand(string $from, string $to, string $command): void`?

Reply:
The variable is an arrangement that saves all commands, executed by the converter, I think it would not be a good idea.

Reply:
But doesn't array always contain ['target_file_extension', 'command']?

Reply:
Done.